### DISCUSSION TITLE
How to restrict area of model generation for out doors

### DISCUSSION THREAD

**Discussion by sunhoro** [NVlabs/instant-ngp#787] [Q&A] [answered]
Hello, does anyone know how to set the boundary for outdoor images or a large project?

For example, if I just run a drone image set of a building, Instant-ngp generates a large cube that includes all the surroundings with ground, sky, tree, etc, and the target building is just somewhere inside the buildings. Which is very hard to navigate due to the heavy model. 

But if you see the provided DroneDeploy Dataset (https://github.com/nickponline/dd-nerf-dataset), only a rectangular shape of the ground+building is rendered. Somehow unnecessary surroundings such as sky and foregrounds are excluded.

1.  Do I need to set the boundary perimeters prior to the training? if so, how can I do it?
2. If the DroneDeploy model is edited after the initial training, how do I export the 3d mesh model?

Thank you!

**Comment by Tom94** [comment, marked as answer]
You can add `"scale"` and `"offset"` (at the outermost scope) to `transforms.json` to accomplish an up- or down-scaling of the dataset at training time. This can reduce reconstruction quality, though -- it might be better to train with all the detail and to then use the "Crop" GUI option to trim down the rendering alone.

As for exporting a 3D model: there's a "mesh it" button in the GUI. (The mesh quality is going to be rather bad for real-world scenes, though. I wouldn't recommend instant-ngp as a go-to model export tool.)

**Reply by sunhoro** [reply]
> You can add `"scale"` and `"offset"` (at the outermost scope) to `transforms.json` to accomplish an up- or down-scaling of the dataset at training time. This can reduce reconstruction quality, though -- it might be better to train with all the detail and to then use the "Crop" GUI option to trim down the rendering alone.
> 
> As for exporting a 3D model: there's a "mesh it" button in the GUI. (The mesh quality is going to be rather bad for real-world scenes, though. I wouldn't recommend instant-ngp as a go-to model export tool.)

Is there any example that I can see how its done? Thank you!
